### ISSUE TITLE
Resize cell sizes based on selected percentage (in top menu)

### ISSUE THREAD

**Issue by ttowntom** [NoroffFEU/yeetsheet#122]
- Create a function that resizes the cells in the sheet, based on the selected percentage selected from the top menu
- Create a event listener that calls the function when a user selects a percentage option

Make sure [this task](https://github.com/NoroffFEU/yeetsheet/issues/113) is done before starting

**Comment by Barvand** [comment]
@BergitTveit @FredrikMohag We encountered an error here. As Team Havørn is working in the header section. The problem we are encountering is that the rendering of the menu has been set up in a certain way that interferes with the drop down menu's in the header section. If we leave our function as is it now, it will interfere with the display of all the drop down menu's in the header. 

This can be fixed if we add some error handling and tweaks to the function. 

I will create a new ticket to implement this feature in the future. 

Anyway, good job @BergitTveit the feature is neat!
